packages/sockethub/src/resource-manager.ts

```diff
@@ -31,7 +31,9 @@ function resourceManagerCycle() {
         }
       }
 
-      if (platformInstance.sessions.size <= 0) {
+      if (platformInstance.global) {
+        // static platform for global use, don't do resource management
```

Comment:
Wouldn't it be better to have a `continue` statement here instead of doing nothing?